packages/ruleset/src/functions/pattern-properties.js

```diff
@@ -5,7 +5,7 @@
 
 const {
   isObject,
-  validateNestedSchemas,
+  validateSubschemas,
```

Comment:
while working on the unevaluatedProperties rule, I discovered that the patternProperties rule should be using validateSubschemas instead of validateNestedSchemas.